### ISSUE TITLE
tugboat stopped working

### ISSUE THREAD

**Issue by mxdpeep** [petems/tugboat#279]
$ ruby --version
```
ruby 2.4.1p111 (2017-03-22 revision 58053) [x86_64-linux]
```

$ tugboat droplets
```
/home/mxdpeep/.rvm/gems/ruby-2.4.1/gems/activesupport-4.0.13/lib/active_support/core_ext/numeric/conversions.rb:121: warning: constant ::Fixnum is deprecated
/home/mxdpeep/.rvm/gems/ruby-2.4.1/gems/activesupport-4.0.13/lib/active_support/core_ext/numeric/conversions.rb:121: warning: constant ::Bignum is deprecated
/home/mxdpeep/.rvm/gems/ruby-2.4.1/gems/activesupport-4.0.13/lib/active_support/core_ext/numeric/conversions.rb:124:in `block (2 levels) in <class:Numeric>': stack level too deep (SystemStackError)
	from /home/mxdpeep/.rvm/gems/ruby-2.4.1/gems/activesupport-4.0.13/lib/active_support/core_ext/numeric/conversions.rb:131:in `block (2 levels) in <class:Numeric>'
	from /home/mxdpeep/.rvm/gems/ruby-2.4.1/gems/activesupport-4.0.13/lib/active_support/core_ext/numeric/conversions.rb:131:in `block (2 levels) in <class:Numeric>'
	from /home/mxdpeep/.rvm/gems/ruby-2.4.1/gems/activesupport-4.0.13/lib/active_support/core_ext/numeric/conversions.rb:131:in `block (2 levels) in <class:Numeric>'
	from /home/mxdpeep/.rvm/gems/ruby-2.4.1/gems/activesupport-4.0.13/lib/active_support/core_ext/numeric/conversions.rb:131:in `block (2 levels) in <class:Numeric>'
	from /home/mxdpeep/.rvm/gems/ruby-2.4.1/gems/activesupport-4.0.13/lib/active_support/core_ext/numeric/conversions.rb:131:in `block (2 levels) in <class:Numeric>'
	from /home/mxdpeep/.rvm/gems/ruby-2.4.1/gems/activesupport-4.0.13/lib/active_support/core_ext/numeric/conversions.rb:131:in `block (2 levels) in <class:Numeric>'
	from /home/mxdpeep/.rvm/gems/ruby-2.4.1/gems/activesupport-4.0.13/lib/active_support/core_ext/numeric/conversions.rb:131:in `block (2 levels) in <class:Numeric>'
	from /home/mxdpeep/.rvm/gems/ruby-2.4.1/gems/activesupport-4.0.13/lib/active_support/core_ext/numeric/conversions.rb:131:in `block (2 levels) in <class:Numeric>'
	 ... 4034 levels...
	from /home/mxdpeep/.rvm/gems/ruby-2.4.1/gems/thor-0.18.1/lib/thor/base.rb:439:in `start'
	from /home/mxdpeep/.rvm/gems/ruby-2.4.1/gems/tugboat-3.0.0/bin/tugboat:10:in `<top (required)>'
	from /home/mxdpeep/.rvm/gems/ruby-2.4.1/bin/tugboat:22:in `load'
	from /home/mxdpeep/.rvm/gems/ruby-2.4.1/bin/tugboat:22:in `<main>'
```

**Comment by petems** [comment]
Ah, I had already fixed this for 2.4.1, but haven't released it yet! https://github.com/petems/tugboat/pull/277

Releasing now as 3.1.0, can you test it for me? 😄

**Comment by mxdpeep** [comment]
fixed!
